content/en/blog/2021/discovery-selectors/index.md

```diff
@@ -0,0 +1,85 @@
+
+## Introduce DiscoverySelectors
+
+In Istio 1.10, we introduced the new `DiscoverySelectors` option to [MeshConfig](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig), which is an array of Kubernetes [selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements). The exact type will be `[]LabelSelector`, as defined [here](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements), allowing both simple selectors and set-based selectors. These selectors apply to labels on namespaces.
+
+You can configure each label selector for expressing a variety of use cases, including but not limited to:
+
+* Arbitrary label names/values, for example, all namespaces with label `istio-discovery=enabled`
+* A list of namespace labels using set-based selectors which carries OR semantics, for example, all namespaces with label `istio-discovery=enabled` OR `region=us-east1`
+* Inclusion and/or exclusion of namespaces, for example, all namespaces with label istio-discovery=enabled AND label key `app` equal to `helloworld`
+
+Note: `DiscoverySelectors` is not a security boundary. Istiod will continue to have access to all namespaces even when you have configured your `DiscoverySelectors`.
+
+## DiscoverySelectors in Action
+
+Assuming you know which namespaces to include as part of the service mesh, as a mesh administrator, you can configure `DiscoverySelectors` at installation time or post-installation by adding your desired discovery selectors to Istio’s MeshConfig resource. For example, you can configure Istio to discover only the namespace that has the label `istio-discovery=enabled`.
+
+1. Enable all namespaces with this label. Using our examples earlier, let’s enable namespace default with label `istio-discovery=enabled`.
+
+{{< text bash >}}
+$ kubectl label namespace default istio-discovery=enabled
+{{< /text >}}
+
+1. Use istioctl to apply the yaml with `DiscoverySelectors` to update your Istio installation. Note, to avoid any impact to your stable environment, we recommend you to use a different revision for your Istio installation:
+
+{{< text bash >}}
+$ istioctl install --skip-confirmation -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+ namespace: istio-system
+spec:
+ # You may override parts of meshconfig by uncommenting the following lines.
+ meshConfig:
+   discoverySelectors:
+     - matchLabels:
+         istio-discovery: enabled
+EOF
+{{< /text >}}
+
+1. Display endpoint configuration from sleep pod:
```

Comment:
from or for? (you have for in the picture title and I like that better) and maybe add `the`

Reply:
updated